ironfish/src/primitives/rawTransaction.ts

```diff
@@ -22,7 +23,7 @@ export class RawTransaction {
   spendingKey = ''
   expiration: number | null = null
   fee = 0n
-  mints: MintDescription[] = []
+  mints: MintData[] = []
```

Comment:
This should just be  `name`, `metadata`, and `value` right? We don't need the other fields in the raw transaction

Reply:
Wrong, we need asset id for validation in post